### PR TITLE
WDC Redfish support for chassis indicator LED toggling.

### PR DESCRIPTION
**Pull request by mikemoerk**
##### SUMMARY
WDC Enclosure support for indicator LED toggling for various chassis components, including:

- Enclosure
- I/O Modules
- Power Supplies
- External Fans
- Internal Fan

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
wdc_redfish_command

##### ADDITIONAL INFORMATION
Sample command:
```
ansible mm4 -m community.general.wdc_redfish_command -a "username={{ user }} password={{ pass }} baseuri={{ioms[0]}} category=Chassis resource_id=FanInternalFRU command=IndicatorLedOn"
```

### CODE REVIEW

File: plugins/modules/remote_management/redfish/wdc_redfish_command.py [hunk 1]
```diff
@@ -54,6 +54,11 @@
       - Timeout in seconds for URL requests to OOB controller.
     default: 10
     type: int
+  resource_id:
+    required: false
+    description:
+      - ID of the component to modify, such as "Enclosure", "IOModuleAFRU", "PowerSupplyBFRU", "FanExternalFRU3", or "FanInternalFRU".
+    type: str
```
Comment (felixfontein) [inline plugins/modules/remote_management/redfish/wdc_redfish_command.py:61]:
```suggestion
    type: str
    version_added: 5.4.0
```

File: plugins/modules/remote_management/redfish/wdc_redfish_command.py [hunk 2]
```diff
@@ -54,6 +54,11 @@
       - Timeout in seconds for URL requests to OOB controller.
     default: 10
     type: int
+  resource_id:
+    required: false
+    description:
+      - ID of the component to modify, such as "Enclosure", "IOModuleAFRU", "PowerSupplyBFRU", "FanExternalFRU3", or "FanInternalFRU".
```
Comment (felixfontein) [inline plugins/modules/remote_management/redfish/wdc_redfish_command.py:60]:
```suggestion
      - ID of the component to modify, such as C(Enclosure), C(IOModuleAFRU), C(PowerSupplyBFRU), C(FanExternalFRU3), or C(FanInternalFRU).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Review by felixfontein — APPROVED** [review]
Looks good to me (without looking too closely at the code, I assume you did enough testing on this).

**Comment by mraineri** [comment]
shipit

**Review by russoz — APPROVED** [review]
LGTM, also assuming thorough tests have been executed

**Comment by felixfontein** [comment]
@mikemoerk thanks for implementing this!
@mraineri @russoz thanks for reviewing this!

**Pull request merged**